hack/lib/common.bash

```diff
@@ -15,16 +15,21 @@ function array.join {
 
 # Loops until duration (car) is exceeded or command (cdr) returns non-zero
 function timeout {
-  local seconds timeout interval
-  interval=5
+  local timeout
+  timeout="${1:?Pass a timeout as arg[1]}"
+  interval="${interval:-1}"
   seconds=0
-  timeout=$1
   shift
-  while eval $*; do
+  ln=' ' logger.debug "${*} : Waiting until non-zero (max ${timeout} sec.)"
+  while (eval "$*" 2>/dev/null); do
```

Comment:
I'm curious if this will print the condition and the result during each execution. It doesn't look like that. I find it useful to see the condition and the actual values when it's evaluated, possibly when running with `-x`

Reply:
This will print the command on stdout just once, but you will see all values in debug log.